README.md

```diff
@@ -84,4 +84,4 @@ The MIT License (MIT). Please see [License File](../LICENSE.md) for more informa
 [link-code-quality]: https://scrutinizer-ci.com/g/GA/BrokerAPI
 [link-downloads]: https://packagist.org/packages/GA/BrokerAPI
 [link-author]: https://github.com/emilianozublena
-[link-contributors]: ../../contributors
+[link-contributors]: https://github.com/asyncapi/asyncapi-php-template/graphs/contributors
```

Comment:
can we update the URL in here? as the name of the template (and repository) slightly changed
https://github.com/asyncapi/php-template